operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,499 @@ spec:
         path: conditions
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes.conditions
-      version: v1alpha1
+      version: v1beta2
+  description: >
+    **Red Hat AMQ Streams** is a massively scalable, distributed, and high performance data streaming platform based on the Apache Kafka® project.
+    AMQ Streams provides an event streaming backbone that allows microservices and other application components to exchange data with extremely high throughput and low latency.
+
+    **The core capabilities include:**
 
+    * A pub/sub messaging model, similar to a traditional enterprise messaging system, in which application components publish and consume events to/from an ordered stream
+
+    * The long term, fault-tolerant storage of events
+
+    * The ability for a consumer to replay streams of events
+
+    * The ability to partition topics for horizontal scalability
+
+    ### CRD Upgrades
+
+    **!!! IMPORTANT !!!** 
+    This release introduces new API version `v1beta2` to all AMQ Streams custom resources.
+    This is a preparation for migration to `apiextensions/v1` which is needed because Kubernetes 1.22 will remove support for `apiextensions/v1beta1`. 
```

Comment:
```suggestion
    This is a preparation for migration to `apiextensions/v1` which is needed because OpenShift 4.9 will remove support for `apiextensions/v1beta1`. 
```